packager/debugger.html

```diff
@@ -14,8 +14,144 @@
+              pointColor: "rgba(113, 188,120,1)",
+              pointStrokeColor: "#fff",
+              data: data
+          }
+      ]
+    };
+
+    var memoryStartingData = {
+      labels: labels,
+      datasets: [
+        {
+          label: "Resident Memory (MB)",
+          fillColor: "rgba(220,220,220,0.2)",
+          strokeColor: "rgba(220,220,220,1)",
+          pointColor: "rgba(220,220,220,1)",
+          pointStrokeColor: "#fff",
+          pointHighlightFill: "#fff",
+          pointHighlightStroke: "rgba(220,220,220,1)",
+          data: data
+        },
+        {
+          label: "Virtual Memory (MB)",
+          fillColor: "rgba(151,187,205,0.2)",
+          strokeColor: "rgba(151,187,205,1)",
+          pointColor: "rgba(151,187,205,1)",
+          pointStrokeColor: "#fff",
+          pointHighlightFill: "#fff",
+          pointHighlightStroke: "rgba(151,187,205,1)",
+          data: data
+        }
+      ]
+    };
+
+
+    var jsCallsStartingData = {
+      labels: labels,
+      datasets: [
+          {
+              fillColor: "rgba(238, 210, 2,0.2)",
+              strokeColor: "rgba(238, 210, 2,1)",
+              pointColor: "rgba(238, 210, 2,1)",
+              pointStrokeColor: "#fff",
+              data: data
+          }
+      ]
+    };
+
+    latestLabel = cpuStartingData.labels[numPoints - 1];
+    // Reduce the animation steps for demo clarity.
+    cpuChart = new Chart(ctx).Line(cpuStartingData, {
+        scaleOverride: true,
+        scaleSteps: 10,
+        scaleStepWidth: 10,
+        scaleStartValue: 0,
+        animation: false
+    });
+
+    memoryChart = new Chart(memoryCtx).Line(memoryStartingData, {
+      animation: false
+    });
+
+    document.getElementById("memory-legend").innerHTML = memoryChart.generateLegend();
+
+    jsCallsChart = new Chart(jsCallsCtx).Bar(jsCallsStartingData, {
+      animation: false
+    });
+});
+
+function updateJSCalls() {
+  var list = [];
+  for (var i = batchedJSCalls.length - 1; i >= 0 ; i--) {
+    // grab a batch of JS Calls
+    var jsCalls = batchedJSCalls[i];
+
+    // display the method called (typically a batch call)
+    var stringItem = "<li>" + jsCalls.moduleMethod;
+
+    // break out the sub js calls
+    var arguments = jsCalls.arguments;
+
+    for (var argument in arguments) {
+      var subCalls = arguments[argument];
+      stringItem += "<ul>";
+      for (var call in subCalls) {
+        var subCall = subCalls[call];
+        // just grab the function name called
+        var subCallFunction = subCall.args[0];
+        // NOTE: there's more debug info that could
+        //       be pulled from the args
+        stringItem += "<li>" + subCall.method +
+            " <em>\"" + subCallFunction + "\"</em>";
+        stringItem += "</li>";
+      }
+      stringItem += "</ul>";
+    }
+
+    stringItem += "</li>";
+    list.push(stringItem);
+  }
+  document.getElementById('js-calls').innerHTML = '<ul>' + list.join('') + '</ul>';
```

Comment:
going to go ahead an merge this anyways - feels much more smooth using React (of course right?? :smiley_cat: )